xcode/Subconscious/Shared/Components/Common/Forms/ValidatedTextField.swift

```diff
@@ -9,15 +9,20 @@ import SwiftUI
 import Combine
 import ObservableStore
 
-struct ValidatedFormField<Output: Equatable, Caption: View>: View {
+enum FieldCaption {
+    case none
+    case text(String)
+}
```

Comment:
Could we use an optional type instead?

Reply:
Yup. `caption` is now of type `String.LocalizationValue?`, forcing us to do the right thing at the callsite.